include/deal.II/base/mpi.h

```diff
@@ -167,6 +171,19 @@ namespace Utilities
               std::vector<T> &sums);
 
     /**
+     * Like the previous function, but take the sums over the elements of a
+     * Vector<number>.
```

Comment:
Use the same template argument in the documentation as in the declaration below (i.e., `number -> T`).